pysmt/test/test_formula.py

```diff
@@ -610,7 +610,7 @@ def test_pickling(self):
 
         self.assertEqual(str(f), "(A & (! B))", str(f))
 
-        serialized = pickle.dumps(f)
+        serialized = pickle.dumps(f, -1)
```

Comment:
This is needed because the use of slots is incompatible with the use of the oldest version of pickle's protocol.